keras_retinanet/bin/debug.py

```diff
@@ -17,6 +17,7 @@
 """
 
 import argparse
+import ntpath
```

Comment:
Prefer to use `os.path`, should be cross platform too.

Reply:
that what i was afraid but indeed, it is cross platform